doc/compiler/runtime/JITHooks.md

```diff
@@ -22,11 +22,49 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 
 # Background
 
-The VM has a mechansim to register callbacks for specific runtime event.
-The JIT makes use of these events to perform tasks either for functional 
-correctness, performance, or housekeeping. 
+The VM has a hook mechanism that is used to perform actions for specific
+runtime events. It also provides the ability to register callbacks for
+these events. Some hooks impact the generation of compiled code, and
+others require the JIT to perform tasks either for functional
+correctness, performance, or housekeeping. This doc outlines the various
+Hooks that are of interest for the JIT.
 
-# Hooks
+# Hooks that impact Compiled Code
+This section outlines various events or Hooks that impact the code that
+the JIT generates.
+
+**IMPORTANT**: The way the compiler tests if one of these
+events can be hooked is by calling `J9HookDisable`; if the return value
+is `0` then the event cannot be hooked, and if the return value is
+**not** `0` then the event **can** be hooked. However, it should be
+pointed out that when `J9HookDisable` returns `0`, it either successfully
+disabled the hook or the hook was already successfully disabled in some
+prior invocation. It is critical that the JIT use this API rather than
```

Comment:
`use` -> `uses`

Reply:
I'm pretty sure `use` is grammatically correct, but after reading the sentence over 20 times, I've confused myself... so I'll make the change since `uses` is also grammatically correct.